src/app/preferences/general-preferences/general-preferences.service.ts

```diff
@@ -1,18 +1,21 @@
 namespace Core {
+  export enum state {
```

Comment:
Don't deplicate the code. Instead, export the enum in the `Page` namespace, and `<reference ...` it to use here.